Adds `tracks` attribute to `Project`, `Topic` and `Challenge` Objects (#84)
Laboratoria/curriculum-parser: README.md

  --version 5.5.0 \
  --lo=./learning-objectives
```

## Known tracks

This project contains an array in [`lib/common.js`](./lib/common.js) called
`knownTracks` which contains the list of tracks that are known by the parser.

> If a new track was ever going to be added to the curriculum, it would need to be

Reviewer: ```suggestion
If a new track is ever going to be added to the curriculum, it must be
```
